hw/ip_templates/ac_range_check/data/ac_range_check.hjson.tpl

```diff
@@ -361,5 +361,32 @@ import math
         ]
       }
     }
+    { name: "ALERT_STATUS",
```

Comment:
Please add this register at the beginning (first register) so that its register address does not depend on the number of configured ranges.